doc/whats_new/v1.0.rst

```diff
@@ -274,6 +274,12 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+:mod:`sklearn.semi_supervised`
+..............................
+
+- |Fix| :meth:`BaseLabelPropagation.fit` now accepts sparse matrices.
```

Comment:
Actually, this is an enhancement and not a fix.
You can use `|Enhancement|`. This entry should be moved in the file `v1.1.rst` now.